src/lessonsExamples/lesson0610/Anagram.java

```diff
@@ -0,0 +1,36 @@
+package lessonsExamples.lesson0610;
+
+import java.util.*;
+
+public class Anagram {
+    private static List<String> VOC = Arrays.asList("anna", "ivan", "naan", "vani", "piotr", "navi");
+
+    public static List<String> getAnagram(String word) {
+        List<String> res = new ArrayList<>();
+        for (String w : VOC) {
```

Comment:
This is a good solution, but we considered a better (faster) one which uses maps.